docs/source/api.rst

```diff
@@ -4,7 +4,7 @@ API Reference
 Dask APIs generally follow from upstream APIs:
 
 -  :doc:`Arrays<array-api>` follows NumPy
--  :doc:`DataFrames <dataframe-api>` follows Pandas
+-  :doc:`DataFrames <dask-expr-api>` follows Pandas
```

Comment:
Maybe we keep the same URL, but change the content?

Reply:
Hm good point, that makes sense